### PR TITLE
GUVNOR-3509: Test marshaling of not matches and not soundslike operators

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins please retest this

File: drools-workbench-models/drools-workbench-models-commons/src/test/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceTest.java [hunk 1]
```diff
@@ -2661,6 +2661,68 @@ public void testConnective() {
...
     @Test
+    public void testNotSoundsLikeAndNotMatches() {
+
+        final String ruleName = "test not soundslike constraint";
+
+        final RuleModel model = new RuleModel();
+        model.name = ruleName;
+
+        final FactPattern pattern = new FactPattern("Person");
+        final CompositeFieldConstraint notMatchesAndNotSounsLike = new CompositeFieldConstraint();
```
Comment (manstis) [inline drools-workbench-models/drools-workbench-models-commons/src/test/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceTest.java]: Type "notMatchesAndNotSound**s**Like"

File: drools-workbench-models/drools-workbench-models-commons/src/test/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceTest.java [hunk 2]
```diff
@@ -2661,6 +2661,68 @@ public void testConnective() {
...
+                + "then"
+                + "end";
+
+        checkMarshalling(expected,
+                         model);
+    }
+
+    @Test
+    public void testNotSoundsLikeAndNotMatchesInDsl() {
+        final String dslDefinition = "There is Person that field not matches and not sounslike {name}";
```
Comment (manstis) [inline drools-workbench-models/drools-workbench-models-commons/src/test/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceTest.java]: Typo "soun**d**slike"

**Review by manstis — COMMENTED** [review]
@jomarko A couple of minor comments.

**Review by manstis — APPROVED** [review]

**Pull request merged**